superset-frontend/src/dashboard/stylesheets/components/chart.less

```diff
@@ -148,3 +148,10 @@
 .time-filter-tabs > .nav-tabs > li > a {
   padding: 4px;
 }
+
+.full-size-style {
+  position: fixed;
+  z-index: 1000;
+  left: 0px;
+  top: 0px;
+};
```

Comment:
a couple nits:
- Name this `full-size`, we know it's applying styling because it's already in css
- Use one of the z-index constants here if one of them works, or add a new one for dashboard modals https://github.com/apache/incubator-superset/blob/7d572d9e9621e6c469d6db0afe650e41fcf297b1/superset-frontend/stylesheets/less/variables.less#L106-L124
- `left` and `top` can just be `0` instead of `0px`
- Remove the semicolon and make sure there's a new line at the end of the file

Reply:
Fixed. 301a8f0c57e8999a76e3185fb659d9723d82598f.
I'm curious: why the new line at the end of the file? @etr2460